packages/terra-popup/tests/nightwatch/components/ArrowPopupClosed.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Popup from '../../../lib/Popup';
+
+class ArrowPopupClosed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  componentDidMount() {
+    this.forceUpdate();
```

Comment:
What is this needed for? Kind of strange for the need to force an update when a component is first mounted